edb/edgeql/ast.py

```diff
@@ -691,6 +691,7 @@ class MigrationBody(DDL):
 class CreateMigration(CreateObject, Migration):
 
     body: MigrationBody
+    script: typing.Optional[str] = None
```

Comment:
This is basically non-normalized `body` of migration? Maybe call it `raw_body`?

Reply:
It is not _necessarily_ non-normalized.  It could be a codegen-ed body too.